main/src/cgeo/geocaching/connector/oc/OkapiClient.java

```diff
@@ -848,6 +862,8 @@ private static void parseCoreCache(final ObjectNode response, @NonNull final Geo
         //set basic properties which are constant / not used for OC platforms
         cache.setPremiumMembersOnly(false);
         cache.setUserModifiedCoords(false);
+
+        DataStore.saveCache(cache, EnumSet.of(SaveFlag.CACHE));
```

Comment:
Why should store the search results in the DB? IMHO, there shouldn't be any direct DB access from individual connectors...

Reply:
Every connector has the DataStore.saveCache in its parse() method:
![image](https://user-images.githubusercontent.com/1258173/180653130-3477b344-0dcf-4353-b481-1bf527cb5b3b.png)

Reply:
Sorry, never mind. Was just wondering why you added it... Thanks for the explanation!